README.md

```diff
@@ -10,3 +10,14 @@ See [EasterEggDetectorSystem](src/main/java/org/terasology/blockdetector/easter/
 ## Credits
 
 [ScannerBeep.ogg](assets/sounds/ScannerBeep.ogg) by kalisemorrison @ [freesound](https://www.freesound.org/people/kalisemorrison/sounds/202530/).
+
+
+# Testing
+
+This module also have tests for the BlockDetectorSystemImpl class. You can find the code [here](src/test/java/org/terasology/blockdetector/systems)
```

Comment:
`also have tests` -> `also has tests`. Add a period (`.`) at the end of sentence.